pkgs/by-name/dc/dc3dd/package.nix

```
@@ -23,6 +32,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   makeFlags = [
     "PREFIX=$out"
+    "CC=cc"
```

```suggestion
    "CC=${stdenv.cc.targetPrefix}cc"
```
this is necessary for cross compilation